packages/core/src/DisabledMixin.js

```diff
@@ -0,0 +1,122 @@
+import { dedupeMixin } from './dedupeMixin.js';
+
+/**
+ * #DisabledMixin
+ *
+ * For now this is a private mixin for option
+ * Do not use it anywhere else - it will become part of field eventually.
+ *
+ * @polymerMixin
+ * @mixinFunction
+ */
+export const DisabledMixin = dedupeMixin(
+  superclass =>
+    // eslint-disable-next-line no-shadow
+    class DisabledMixin extends superclass {
+      static get properties() {
+        return {
+          disabled: {
+            type: Boolean,
+            reflect: true,
+          },
+          // we us a property here as if we use the native tabIndex we can not set a default value
```

Comment:
'us' => 'use'

https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker 😉

Reply:
I have it installed... but us is a correct word and it did not mark it as "wrong" 🙈

Reply:
changed 👍